content/en/profiler/enabling/dotnet.md

```diff
@@ -64,6 +64,7 @@ The following profiling features are available in the following minimum versions
 | [Code Hotspots][12]        | 2.7.0+                       | All supported runtime versions.      |
 | [Endpoint Profiling][13]            | 2.15.0+                       | All supported runtime versions.      |
 | Timeline            | 2.30.0+                       | All supported runtime versions (except .NET 5+ required for garbage collection details).     |
+| Azure App Serivces (public beta) | 2.39.0+                       | All supported runtime versions.   |
```

Comment:
Should we consider Windows and Linux AAS support starting from 2.39 ?
For Linux, changes to support AAS will be part of the 2.39 but for Windows, since it depends on the packaging and we do not know for sure since when the profiler is available on AAS.

Reply:
Let's keep 2.39. Better to not mislead or become verbose (and current customers for who it already works on Windows will likely not upgrade because of the doc anyway)